front/src/components/Table/components/SettingsIcon/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { IconButton, Menu } from "@material-ui/core";
+import SettingsIcon from "@material-ui/icons/Settings";
+import { useDispatch } from "react-redux";
+
+import { deleteItem } from "features/books/booksSlice";
+
+import { useStyles, MenuItem } from "./styles";
+
+interface Props {
+  id: number;
+}
+
+const MenuAppBar: React.FC<Props> = (id) => {
+  const dispatch = useDispatch();
+
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const classes = useStyles();
+
+  const handleMenu = (event: any) => {
```

Comment:
mozesz z tego i handleClose zrobic jedna funkcje, ktora przyjmie event?: React.MouseEvent<HTMLDivElement, MouseEvent>

setAnchorEl(event ? event.currentTarget : null)

mozliwe ze sa troche inne typym, na szybko mysle

Reply:
wkleilem z podpowiedzi z hovera ale nie dzala to wgl.
Skipped.